models/_sector/dex/trades/arbitrum/dex_arbitrum_base_trades.sql

```diff
@@ -53,6 +53,10 @@ WITH base_union AS (
             , project_contract_address
             , tx_hash
             , evt_index
+            , false AS override_bought_decimals
+            , false AS override_sold_decimals
+            , false AS swap_bought_decimals
+            , false AS swap_sold_decimals
```

Comment:
I wonder if we can have better approach for these.

Reply:
I know it's not the most elegant solution, but as defaulting of the decimals to 18 or swapping between bought and sold token decimals goes all the way down to be used in `enrich` macro - I couldn't think of any better solution without touching `base_trades` models.

I'm open to suggestions.